src/languages/es-ES.ts

```diff
@@ -328,8 +328,10 @@ export default class extends Language {
 		INHIBITOR_MUSIC_BOT_VOICE_CHANNEL: `Temo que necesito estar en un canal de voz antes de poder usar este comando, ¡por favor muéstreme el camino!`,
 		INHIBITOR_MUSIC_BOTH_VOICE_CHANNEL: `¡Hey! Parece que no estamos en el mismo canal de voz, ¡por favor únete conmigo!`,
 
+		MUSICMANAGER_FETCH_NO_ARGUMENTS: `I need you to give me the name of a song!`,
```

Comment:
```suggestion
		MUSICMANAGER_FETCH_NO_ARGUMENTS: '¡Necesito que me des el nombre de una canción!',
```

Also removed the backticks since they are not needed for this one.